go.mod

```diff
@@ -5,19 +5,19 @@ go 1.16
 require (
 	github.com/aws/aws-sdk-go v1.37.4
 	github.com/aws/aws-sdk-go-v2 v0.23.0
-	github.com/crossplane/crossplane-runtime v0.14.1-0.20210722005935-0b469fcc77cd
+	github.com/crossplane/crossplane-runtime v0.15.1-0.20210907213605-9779b31a42ce
 	github.com/crossplane/crossplane-tools v0.0.0-20210320162312-1baca298c527
 	github.com/evanphx/json-patch v4.11.0+incompatible
 	github.com/go-ini/ini v1.46.0
-	github.com/google/go-cmp v0.5.5
+	github.com/google/go-cmp v0.5.6
 	github.com/mitchellh/copystructure v1.0.0
-	github.com/onsi/gomega v1.13.0
+	github.com/onsi/gomega v1.14.0
 	github.com/pkg/errors v0.9.1
 	github.com/stretchr/testify v1.7.0
 	gopkg.in/alecthomas/kingpin.v2 v2.2.6
-	k8s.io/api v0.21.2
-	k8s.io/apimachinery v0.21.2
-	k8s.io/client-go v0.21.2
-	sigs.k8s.io/controller-runtime v0.9.2
-	sigs.k8s.io/controller-tools v0.4.0
+	k8s.io/api v0.21.3
+	k8s.io/apimachinery v0.21.3
+	k8s.io/client-go v0.21.3
+	sigs.k8s.io/controller-runtime v0.9.6
```

Comment:
I suspect this will require rebasing lots of the other "new resource" PRs in this repo, and possibly the two large bumps of AWS SDK PRs? If so, I'm a bit worried about contributor "churn" due to this. The "solution" is probably to reduce the number of WIP PRs, but it's also a question of what goes in first of these changes (which ones are mechanical and easier to rebase).

Reply:
Fair point. We already released patches with this fix, so I'm happy to hold off and take the rebase hit after we get the backlog down a bit.